src/include/execution/sql/runtime_types.h

```diff
@@ -431,7 +446,17 @@ class EXPORT Decimal {
    * @param seed The value to seed the hash with.
    * @return The hash value for this decimal instance.
    */
-  hash_t Hash(const hash_t seed) const { return common::HashUtil::HashCrc(value_); }
+  hash_t Hash(const hash_t seed) const {
+    uint128_t x = value_;
+    const uint64_t k_mul = 0x9ddfea08eb382d69ULL;
```

Comment:
can you document where this comes from and why we do this?

Reply:
in the PR